desktop/packages/linux/rs.ruffle.Ruffle.desktop

```diff
@@ -56,6 +59,7 @@ Keywords[ja]=flash;swf;プレイヤー;エミュレーター;rust
 Keywords[ko]=플래시;swf;플레이어;에뮬레이터;rust
 Keywords[nl]=flash;swf;speler;emulator;rust
 Keywords[pl]=flash;swf;odtwarzacz;emulator;rust
+Keywords[pt]=flash;swf;player;emulador;rust
```

Comment:
I guess it should be `player` -> `reprodutor` assuming the above is correct. Similarly in other places with keywords

Reply:
This is now "leitor", as explained in https://github.com/ruffle-rs/ruffle/pull/18426#issuecomment-2452470579.